### PR TITLE
Footer static pages

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

File: newsroom/public/views.py [hunk 1]
```diff
@@ -2,6 +2,12 @@
 from newsroom.public import blueprint
 
 
-@blueprint.route('/copyright')
-def copyright():
-    return flask.render_template('copyright.html')
+@blueprint.route('/privacy')
+def privacy():
+    return flask.render_template('privacy.html')
+@blueprint.route('/terms')
```
Comment (akintolga) [inline newsroom/public/views.py]: I think you need 2 blank spaces before @blueprint otherwise flake8 will faill

File: theme/footer.html [hunk 2]
```diff
@@ -3,9 +3,8 @@
         <img src="{{ url_for('theme', filename = 'login-logo-footer.svg') }}" class="d-none d-lg-block mr-3" />
     </div>
     <div>
-        {% if config.PRIVACY_POLICY %}<a href="{{ config.PRIVACY_POLICY }}" target="_blank">{{ gettext('Privacy Policy') }}</a>{% endif %}
-        {% if config.TERMS_AND_CONDITIONS %}<a href="{{ config.TERMS_AND_CONDITIONS }}" target="_blank">{{ gettext('Terms and Conditions') }}</a>{% endif %}
-        {% if config.CONTACT_ADDRESS %}<a href="{{ config.CONTACT_ADDRESS }}" target="_blank">{{ gettext('Contact Us') }}</a>{% endif %}
-        {% if config.SHOW_COPYRIGHT %}<a href="{{ url_for('public.copyright') }}" target="_blank">{{ gettext('Copyright') }}</a>{% endif %}
+        {% if config.SHOW_PRIVACY_POLICY %}<a href="{{ url_for('public.privacy') }}" target="_blank">{{ gettext('Privacy Policy') }}</a>{% endif %}
+        {% if config.SHOW_TERMS %}<a href="{{ url_for('public.terms') }}" target="_blank">{{ gettext('Terms and Conditions') }}</a>{% endif %}
+        {% if config.SHOW_CONTACT %}<a href="{{ url_for('public.contact') }}" target="_blank">{{ gettext('Contact Us') }}</a>{% endif %}
```
Comment (petrjasek) [inline theme/footer.html]: I guess you don't need those if's, that's for configuring core template, but this one is custom anyhow

Comment (darconny) [reply theme/footer.html]: Thanks, OK I will remove if's and leave only <a> tags.

File: newsroom/public/views.py (hunk 1 above)
Comment (darconny) [reply newsroom/public/views.py]: Thanks I will try adding 2 blank spaces before @blueprint.

File: newsroom/public/views.py [hunk 3]
```diff
@@ -2,6 +2,12 @@
 from newsroom.public import blueprint
 
 
-@blueprint.route('/copyright')
-def copyright():
-    return flask.render_template('copyright.html')
+  @blueprint.route('/privacy')
```
Comment (petrjasek) [inline newsroom/public/views.py]: this should be like:
```py
@blueprint.route('/privacy')
def privacy():
    return flask.render_template('privacy.html')


@blueprint.route('/terms')
def terms():
    return flask.render_template('terms.html')


@blueprint.route('/contact')
def contact():
    return flask.render_template('contact.html')
```
so 2 empty lines before each `@blueprint`

**Pull request merged**